service/internal/telemetry/process_telemetry.go

```diff
@@ -159,7 +159,11 @@ func (pm *processMetrics) updateCPUSeconds() float64 {
 		return 0
 	}
 
-	return times.Total()
+	total := times.User + times.System + times.Idle + times.Nice +
+		times.Iowait + times.Irq + times.Softirq + times.Steal + times.Guest +
+		times.GuestNice
```

Comment:
Looking at https://github.com/shirou/gopsutil/pull/1325 and https://www.opsdash.com/blog/cpu-usage-linux.html, I wonder if we shouldn't remove `Guest` and `GuestNice`.

Reply:
Oh! That is a really good point! Thanks for sharing that article.
Probably `gopsutil` should remove them too, right? Since:

> Guest and Guest Nice: The process (a hypervisor) is running a virtual CPU. These numbers are already included in User and Nice.

Reply:
They do it for `getAllBusy`. But I suppose they don't want to change the legacy behavior of `Total`.